tools/clang/test/CodeGenSPIRV_Lit/decoration.no-contraction.stage-vars.hlsl

```diff
@@ -18,15 +18,17 @@
 // CHECK-NEXT: OpDecorate [[cxcy_1:%[0-9]+]] NoContraction
 // CHECK-NEXT: OpDecorate [[cxcy_plus_dz_1:%[0-9]+]] NoContraction
 
-// CHECK-NOT: OpDecorate [[cxcy_2]] NoContraction
-// CHECK-NOT: OpDecorate [[cxcy_plus_dz_2]] NoContraction
+// IMPLICIT-CHECK-NOT: OpDecorate [[cxcy_2]] NoContraction
```

Comment:
Does ’--implicit-check-not’ need to be passed on the command line for this?

Reply:
No... I didn't know that existed, but I can use it instead and it's probably more clear. You know an API is good when you make up a name for something and it actually already exists with that exact name.